discretisedfield/field.py

```diff
@@ -444,3 +447,48 @@ def _writeovf(self, filename, representation="txt"):
         f.write(footerstr)
 
         f.close()
+
+
+    def _get_colormap(self, name, n):
+        data = []
+        cmap = pylab.cm.get_cmap(name, n)
+        for i in range(cmap.N):
+            rgb = cmap(i)[:3]
+            data.append(rgb)
+
+        return np.array(data)
+
+
+    def plot3d(self, vector_scale=1.0, direction=0, color=0xff, colormap_name='', **kwargs):
+
```

Comment:
We need a documentation string, please.

Reply:
It seems like this function was accidentally committed to this branch (and this pull request). This function was drafted as an exercise when @sergii-mamedov and @marijanbeg were working together. Its implementation and interface are part of Task 3 from the backlog and documentation is Task 5. In this pull request, I would like to stay concentrated on Tasks 2 and 6.